UM/Mesh/MeshData.py

```diff
@@ -228,6 +228,15 @@ def reserveFaceCount(self, num_faces):
 
         self._vertex_count = 0
         self._face_count = 0
+
+
+    def setIndices(self, indices):
+        self._indices = indices
+        self._face_count = int(self._indices.size / 3)
+
+    def setVertices(self, vertices):
+        self._vertices = vertices
+        self._vertex_count = int(self._vertices.size / 3)
```

Comment:
And you might want to consider resetting the normals and indices in this function. Up to your discretion though, because that requires you to first call `setVertices` and then `setIndices` in that order (as we are doing now).